coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/loader/sql/messages/FromProtobuf.scala

```diff
@@ -55,7 +55,7 @@ object FromProtobuf {
        nextCounterValue = di.nextCounterValue,
        localeName = di.localeName,
        obfuscationKey = di.obfuscationKey.toByteArray,
-       resourceName = di.resourceName
+       resourceName = DatasetResourceName(di.resourceName.getOrElse(sys.error("Dataset info without resource name")))
```

Comment:
Does `sys.error` explode? Or at least not return a String?

Reply:
Yes, it explodes.